src/data_helper.py

```diff
@@ -144,10 +151,16 @@ def get_prediction_generator(self, batch_size):
                     img = Image.open(self.X_test_filename[start_offset + j])
                     img.thumbnail(self.img_resize)
 
-                    # Augment the image `img` here
-
                     # Convert to RGB and normalize
-                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32) / 255
+                    img_array = np.asarray(img.convert("RGB"), dtype=np.float32)
+
+                    img_array = img_array[:, :, ::-1]
+                    # Zero-center by mean pixel
+                    img_array[:, :, 0] -= 103.939
+                    img_array[:, :, 1] -= 116.779
+                    img_array[:, :, 2] -= 123.68
+                    img_array = img_array / 255
```

Comment:
I have read that its still a good idea to have all image arrays as smaller numbers so this is why we divide by 255 after subtracting the mean.